src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -518,6 +524,26 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
   }
 
   /**
+    *  Publish workflow structure to centralized logging.
+    */
+  private def publishWorkflowEvent() : Unit = {
+
+    import org.json4s._
+    import org.json4s.native.Serialization.{write}
+
+    logger.info(s"--------> publishWorkflowEvent ")
+    implicit val formats = DefaultFormats + FieldSerializer[WorkflowExecution]()
+    implicit val actorSystem = context.system;
+
+    for{
+      status <-globalDataAccess.getWorkflowExecution(workflow.id)
+    } yield {
```

Comment:
No need for the braces on the yield closure. Pull the expression on to this line. Also fix the spacing issue in the expression